composer.json

```diff
@@ -14,7 +14,7 @@
         "behat/mink-selenium2-driver": "1.4.x-dev as 1.3.x-dev",
         "composer/installers": "~1.5",
         "drupal-composer/drupal-scaffold": "~2.2",
-        "drupal/address": "^1.8",
+        "drupal/address": "^1.9",
```

Comment:
Was the update really needed? Isn't enough to just fix the tests?

Reply:
Replied [here](https://github.com/openeuropa/oe_translation/pull/118#discussion_r527556941).